helper/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "helper"
```

Comment:
maybe something like `ipa-mpc-helper`? Or brainstorm for a better name?

Reply:
+1 on `ipa-mpc-helper`. Or we could remove "ipa" and make them just `macros` and `mpc-helper`.

Reply:
it seems to be common to keep the prefix: https://github.com/rust-random/rand

Reply:
and maybe ipa should become `ipa-core`